frontend/styles/components/campaigns/dashboard/CampaignDashboardStyles.ts

```diff
@@ -65,11 +65,14 @@ export const couponsTableContainerSx: SxProps = {
   overflow: 'auto',
 };
 
-export const campaignImageSx: SxProps = {
+export const campaignImageStackSx: SxProps = {
   width: '100%',
   height: '100%',
-  minHeight: '100%',
-  maxHeight: '100%',
+  backgroundColor: '#F4F4F4',
+};
+
+export const campaignImageSx: SxProps = {
```

Comment:
If you change a style used by other components, you should check that the other components are not affected. Right now this spoils the admin campaign dashboard layout